src/videojs-contrib-hls.js

```diff
@@ -278,8 +279,42 @@ export default class HlsHandler extends Component {
       let seekable;
 
       if (!updatedPlaylist) {
+        let selectedPlaylist = this.selectPlaylist();
+
+        if (!selectedPlaylist.attributes.AUDIO) {
+          // from safari defaults
+          this.tech_.audioTracks().addTrack(new AudioTrack({
+            enabled: true,
+            id: '1',
+            kind: 'main',
+            tech: this.tech_
```

Comment:
It's not clear to me why it should be. If the tech owns the audioTracks and the audioTracks is called to add a track, it seems like the audio tracks could pass along the tech instead of requiring it from the caller.